infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/BaseSnapshotReader.java

```diff
@@ -45,15 +46,15 @@ public abstract class BaseSnapshotReader extends SnapshotReader {
      */
     private final int maxDataSizePerMsg;
     private final Optional<DistributionSummary> messageSizeDistributionSummary;
-    private final CorfuRuntime rt;
-    private long snapshotTimestamp;
-    protected Set<String> streams;
-    private PriorityQueue<String> streamsToSend;
-    private long preMsgTs;
-    private long currentMsgTs;
-    private OpaqueStreamIterator currentStreamInfo;
-    private long sequence;
-    private OpaqueEntry lastEntry = null;
+    final CorfuRuntime rt;
```

Comment:
They are no longer private member variables?

Reply:
inheritor now needs to access it